src/CardDeck.php

```diff
@@ -103,6 +103,7 @@ protected function init(): void
 
         if ($this->paginator !== false) {
             $this->addPaginator();
+            $this->issetApp() ? $this->getApp()->stickyGet($this->paginator->name) : $this->stickyGet($this->paginator->name);
```

Comment:
this seems wrong to me, we do not want any app/global sticky

Reply:
Same principle as for Grid paginator. Since Paginator is added outside the grid or card deck, we need a way to remember which page a user has click prior to run the action and get back to that same page after the action finishes its execution.

Reply:
That is correct reasoning, but incorrect implementation - Paginator must be somehow bounded with Grid.

Reply:
Agree with you on this one. But implementation changes are for another PR.

Reply:
With `mergeStickyArgsFromChildView` it was very easily possible. Now I think you need to code someting simillar, let's decise before https://github.com/atk4/ui/pull/1651

Reply:
This is fixed now. Did not realize it at first but the error was somewhere else.